test/postgresql.discover.test.js

```diff
@@ -257,6 +288,33 @@ describe('Discover model primary keys', function() {
     });
   });
 
+  it('primary key should be discovered, and db generates instances properly', function(done) {
+    db.discoverPrimaryKeys('demo', function(err, models) {
+      if (err) {
+        console.error(err);
+        done(err);
+      } else {
+        models.forEach(function(m) {
+          assert.deepEqual(m, {owner: 'public',
+            tableName: 'demo',
+            columnName: 'demoId',
+            keySeq: 1,
+            pkName: 'demo_pkey'});
+        });
+      }
+    });
+    Demo.create({id: 1, name: 'checkCase'}, function(err) {
+      should.not.exists(err);
+      Demo.findOne({}, function(err, d) {
+        should.not.exists(err);
+        should.exists(d);
```

Comment:
Similar thing here: `should.exists(d);` ->  `should.exist(d);` (https://shouldjs.github.io/#should-exist)